data/transport missions.txt

```diff
@@ -1412,6 +1412,137 @@ mission "Hallucination"
+	passengers 2
+		illegal 150000 `Your passengers turn out to be wanted criminals. As they are hauled off in chains, you vehemently protest that you knew nothing of their shady histories. The authorities threaten you with jail time, and you quickly contact a lawyer. You wind up quite a bit poorer through a combination of fines, bribes, and attorney's fees (the breakdown remains somewhat unclear).`
+		stealth
+
+	to offer
+		random < 8
+		"rejected illegal jobs" < 0
+		has "Shady passenger transport 1: done"
+
+	on offer
+		dialog `As you are walking through the spaceport, two men in nondescript clothing appear out of nowhere and offer <payment> for discreet, no-questions-asked passage to <destination>.`
+
+	on accept
+		dialog `You quickly bundle the men onto your ship before anyone sees them. As you're helping unpack their luggage, one of them casually mentions that they may have attracted the attention of some bounty hunters, and would prefer not to be captured. You should probably avoid being scanned, too."`
+		"rejected illegal jobs" --
+
+	on decline
+		"rejected illegal jobs" ++
+
+	on complete
+		payment 100000
+		dialog `Your passengers depart after paying you the agreed-upon sum of <payment>, off to make new lives for themselves doing who-knows-what.`
+
+	npc
+		government "Bounty Hunter"
+		personality nemesis plunders waiting
+		system
+			distance 6 9
+		fleet "Bounty Hunters"
+
+
+
+mission "Shady passenger transport 3"
+	name "Shady passenger transport"
+	description "Discreetly transport <bunks> passengers to <destination>. If you are caught with them, the legal consequences may be severe."
+	minor
+	repeat
+	source
+		attributes frontier "dirt belt" rim north south pirate
+	destination
+		attributes "near earth" core urban
+		distance 5 10
+	passengers 4 13
+		illegal 400000 `It's rather difficult to feign ignorance of your passengers' background during your questioning. You wind up having to pay a small fortune in bribes to keep yourself from being charged as an accomplice very serious-sounding crimes. Even so, you fear that your legal status may have suffered a blow...`
+		stealth
+
+	to offer
+		random < 5
+		"rejected illegal jobs" < -5
+		has "Shady passenger transport 2: done"
+
+	on offer
+		dialog `A small group of fearsome-looking young men and women brazenly walk up to you in broad daylight, explaining that they're in some legal trouble and need to get to <destination>, offering <payment> to secure discreet passage. You must be getting quite a reputation in certain circles. Transporting them might be very dangerous. On the other hand, they're offering a small fortune...`
+
+	on accept
+		dialog `the mercenaries (or gang members, or fleeing bank robbers, or who knows what else) make themselves at home on your ship, and begin drinking and arguing. You wonder what you've gotten yourself into...`
+		"rejected illegal jobs" --
+
+	on decline
+		"rejected illegal jobs" ++
+
+	on fail
+		"reputation: Republic" = -1
```

Comment:
Unfortunately this will punish a player who takes this job and then has a moral change-of-heart and aborts it. Without special `on abort` handling, I'm not sure this should be added.

Also, rep changes should use min/max statements, e.g. `<?=` to avoid free rep boosts.

Reply:
Good point. We should probably look into adding an optional way to cause reputation hits as part of the illegal or stealth tag